src/libs/CurrencyUtils.ts

```diff
@@ -87,20 +87,22 @@ function convertToBackendAmount(amountAsFloat: number): number {
  *
  * @note we do not support any currencies with more than two decimal places.
  */
-function convertToFrontendAmountAsInteger(amountAsInt: number): number {
-    return Math.trunc(amountAsInt) / 100.0;
+function convertToFrontendAmountAsInteger(amountAsInt: number, currency: string = CONST.CURRENCY.USD): number {
+    const decimals = getCurrencyDecimals(currency);
+    return Number((Math.trunc(amountAsInt) / 100.0).toFixed(decimals));
 }
 
 /**
  * Takes an amount in "cents" as an integer and converts it to a string amount used in the frontend.
  *
  * @note we do not support any currencies with more than two decimal places.
  */
-function convertToFrontendAmountAsString(amountAsInt: number | null | undefined): string {
+function convertToFrontendAmountAsString(amountAsInt: number | null | undefined, currency: string = CONST.CURRENCY.USD): string {
     if (amountAsInt === null || amountAsInt === undefined) {
         return '';
     }
-    return convertToFrontendAmountAsInteger(amountAsInt).toFixed(2);
+    const decimals = getCurrencyDecimals(currency);
+    return convertToFrontendAmountAsInteger(amountAsInt, currency).toFixed(decimals);
```

Comment:
Why we need to use toFixed(decimals) here? we just used toFixed(decimals) in convertToFrontendAmountAsInteger

Reply:
@ShridharGoel How about this?

Reply:
I think we should use toString method

Reply:
yeah, make sense